app/controllers/rescue_requests_controller.rb

```diff
@@ -3,15 +3,43 @@ def new
   end
 
   def create
-    render plain: Disaster.find(params[:disaster_id]).rescue_requests.create(lat: params[:lat], long: params[:long]).id
+    request = Disaster.find(params[:disaster_id]).rescue_requests.create(lat: params[:lat], long: params[:long])
+    queue_assess request
+    render plain: request.id
   end
 
   def update_short
-    RescueRequest.find(params[:person_id]).update(params.permit(RescueRequest.column_names).to_h)
+    request = RescueRequest.find(params[:person_id])
+    request.update(params.permit(RescueRequest.column_names).to_h)
```

Comment:
`params.permit(RescueRequest.column_names)` is a Bad Idea. You really don't want someone to be able to update the record `id` or `disaster_id` or `key`, etc. This should at least be the params-selection approach I took [here](https://github.com/CrowdRescueHQ/CrowdRescue/blob/master/app/controllers/rescue_requests_controller.rb#L38), and preferably full strong params.